scripts/apps/stream/services/index.ts

```diff
@@ -1,11 +1,13 @@
-ActivityMessageService.$inject = ['gettextCatalog'];
-export function ActivityMessageService(gettextCatalog) {
+import {gettext} from 'core/ui/components/utils';
+
+ActivityMessageService.$inject = [];
+export function ActivityMessageService() {
     return {
         format: (activity) => {
             let displayMessage;
 
             if (activity.name !== 'notify') {
-                displayMessage = gettextCatalog.getString(activity.message);
+                displayMessage = gettext(activity.message);
```

Comment:
Do not use `gettext` for dynamic values. Instead an alias of `gettext` should be used so the code inside `gettext` isn't extracted and sent to Transifex.

It doesn't look like it makes sense to translate this at all.